### ISSUE TITLE
new build to NPM

### ISSUE THREAD

**Issue by mectechx** [phoinixi/jsonresume-theme-stackoverflow#84]
Hello,

Is the most recent release on https://www.npmjs.com/package/jsonresume-theme-stackoverflow up to date with this code? I notice an issue with date formatting in the Volunteer section on the registry, but it appears to be MY formatted correctly in the code. 
https://github.com/mectechx/jsonresume-theme-stackoverflow/blob/0c77917f4c5eadfa50d600722c773a5366b174ea/theme/partials/volunteer.hbs#L19

In the brief screenshot, the above is the Work section, and the below date is the Volunteer section. The dates should be formatted the same

TLDR Please update NPM package if not already matching master/HEAD
<img width="198" alt="Screenshot 2024-02-06 at 3 23 00 PM" src="https://github.com/phoinixi/jsonresume-theme-stackoverflow/assets/117760303/2d82e1f6-5ae4-4310-9125-62fd98e492a5">

**Comment by phoinixi** [comment]
done
